cpp_exercises/day2/date/tests/tst_date.cpp

```diff
@@ -134,6 +135,16 @@ void testDate::equalityOperatorTest() {
     QVERIFY(dateA == dateB);
 }
 
+void testDate::addYearsTest() {
+    Date date(26, Date::October, 2020);
+    date.addYears(13);
+    QCOMPARE(date, Date(26, Date::October, 2033));
+    date.addYears(-13);
+    QCOMPARE(date, Date(26, Date::October, 2020));
+    date.addYears(-2021);
+    QCOMPARE(date, Date(26, Date::October, 0));
+}
```

Comment:
Cosa dovrebbe succedere se provo ad aggiungere anni in modo da causare l'overflow dell'anno?
Cosa dovrebbe succedere se provo ad aggiungere 4 anni a partire dal 29 febbraio di un anno bisestile?
Cosa dovrebbe succedere se provo ad aggiungere 1 anno a partire dal 29 febbraio di un anno bisestile?

Reply:
Inserito clamp del giorno quando si passa dal 29 febbraio di un anno bisestile ad uno non bisestile.

Reply:
@lupiz-0 mi raccomando, ricordati di pushare i commit